File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/entity/PhoneNumber.java
package org.nirvanasms.api.core.entity;

public class PhoneNumber {
	private String number;
	private boolean isValid;
	
	public PhoneNumber(){}
	
	public PhoneNumber(String phoneNumber){
		this.number = phoneNumber;
	}

	public String getNumber() {
		return number;
	}

	public void setNumber(String number) {
		this.number = number;
	}

	public boolean isValid() {
		return isValid;
	}

	public void setValid(boolean isValid) {
		this.isValid = isValid;
	}
}

File: NirvanaSMS.Api.CSharp/LibraryProject/Core/Entity/PhoneNumber.cs
namespace NirvanaSMS.Api.Core.Entity
{
    public class PhoneNumber
    {
        public string Number { get; set; }
        public bool IsValid { get; set; }

        public PhoneNumber()
        {

        }

        public PhoneNumber(string phoneNumber)
        {
            Number = phoneNumber; 
        }
    }
}
